scss/cayman.scss

```diff
@@ -1,6 +1,10 @@
 $large-breakpoint: 64em;
 $medium-breakpoint: 42em;
 
+@mixin smooth-scroll {
+  -webkit-overflow-scrolling: touch;
+}
```

Comment:
Since there is just one property here, let's kill the mixin and use `-webkit-overflow-scrolling` directly below.